### PR TITLE
fix(space): ECM files could be attached and will be displayed as link…

### PR DESCRIPTION
**Pull request by akoushke**
…s for now

### CODE REVIEW

File: packages/node_modules/@ciscospark/react-component-activity-item/src/index.js [hunk 1]
```diff
@@ -38,6 +39,23 @@ const defaultProps = {
   isAdditional: false
 };
 
+function createECMItemComponents(activity, props) {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-activity-item/src/index.js]: can we add some jsdoc?

File: packages/node_modules/@ciscospark/react-component-activity-item/src/index.js [hunk 2]
```diff
@@ -38,6 +39,23 @@ const defaultProps = {
...
+  const links = [];
+
+  for (const item of activity.links.items) {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-activity-item/src/index.js]: Let's verify that `links` and `links.items` exist so it doesn't crash the widget if the data changes on us again.

File: packages/node_modules/@ciscospark/react-component-activity-item/src/index.js [hunk 3]
```diff
@@ -58,14 +90,20 @@ function ActivityItem(props) {
...
-      <ActivityShareFiles
-        content={activity.content}
-        displayName={activity.displayName}
-        files={activity.files.items}
-        {...props}
-      />
-    );
+    if (activity.contentCategory === LINKS_CONTENT_CATEGORY) {
+      // ECM files will be displayed as links for now
+      itemComponent = createECMItemComponents(props);
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-component-activity-item/src/index.js]: It might be worthwhile to make this its own react component so that we can style it properly once we get UI info on how this is supposed to look. Not a blocker though.

File: package.json [hunk 4]
```diff
@@ -58,23 +58,23 @@
...
+    "@ciscospark/internal-plugin-team": "1.47.3",
+    "@ciscospark/internal-plugin-user": "1.47.3",
+    "@ciscospark/plugin-authorization": "1.47.3",
+    "@ciscospark/plugin-logger": "1.47.3",
+    "@ciscospark/plugin-people": "1.47.3",
+    "@ciscospark/plugin-phone": "1.47.3",
+    "@ciscospark/spark-core": "1.47.3",
     "@ciscospark/storage-adapter-local-forage": "1.40.1",
-    "@ciscospark/storage-adapter-local-storage": "1.43.2",
+    "@ciscospark/storage-adapter-local-storage": "1.47.3",
```
Comment (moriahmaney) [inline package.json]: can we split the version update to a different commit for the changelog please and thanks?

File: packages/node_modules/@ciscospark/react-component-activity-item/src/index.js [hunk 5]
```diff
@@ -38,6 +40,36 @@ const defaultProps = {
   isAdditional: false
 };
 
+/**
+ * It will store the links and the names of the file and display them
+ * as link componetns seperated by commas
```
Comment (moriahmaney) [inline packages/node_modules/@ciscospark/react-component-activity-item/src/index.js]: components* and separated* 

not blocking though.
